### DISCUSSION TITLE
Zwrotne resetowanie hasła

### DISCUSSION THREAD

**Discussion by abelino2** [AzureCommunityPL/group#345] [Pomoc] [answered]
Chciałabym mieć możliwość resetowania haseł za pomocą sms, które są synchronizowane z lokalnego serwera do chmury za pomocą AD Connect. Proszę o informację jak nazywa się taka licencja.

**Comment by eklime** [comment, marked as answer]
Proszę: https://docs.microsoft.com/en-us/azure/active-directory/authentication/tutorial-enable-sspr-writeback
